src/components/monitor/ActiveDelegates.vue

```diff
@@ -49,7 +49,7 @@
 
     <table-column show="approval" :label="$t('Approval')" header-class="right-header-cell pr-5 md:pr-10 hidden md:table-cell" cell-class="py-3 px-4 md:pr-10 text-right border-none hidden md:table-cell">
       <template slot-scope="row">
-        <span v-tooltip="{ content: readableCrypto(row.vote, true, 2), placement: 'left' }">
+        <span v-tooltip="{ content: readableCrypto(row.vote, true, 2), placement: 'top' }">
```

Comment:
`placement: 'top'` is the default value :)

Reply:
doesn't really hurt to specify that in the code but would change that for sure if it's wanted :)

Reply:
It's fine as I soon will go through the code with a broom anyways :trollface: